va_explorer/va_analytics/utils/plotting.py

```diff
@@ -350,204 +365,276 @@ def get_pivot_counts(va_df, index_col, factor_col):
     return counts
 
 
-def cod_group_plot(va_df, cod_groups=[], demographic="overall", N=10, height=None, vertical_spacing=.15,\
-                   chosen_cod="all"):
+def cod_group_plot(
+    va_df,
+    cod_groups,
+    demographic="overall",
+    n=10,
+    height=None,
+    vertical_spacing=0.15,
+    chosen_cod="all",
+):
     figure = go.Figure()
-    # if no demographic chosen (i.e. overall), color-code by cause-of-death group. Otherwise, color-code by demographic
+    cod_groups if cod_groups else []
+
+    # if no demographic chosen (i.e. overall), color-code by cause-of-death
+    # group. Otherwise, color-code by demographic
     demographic = demographic.lower()
-    if demographic in ['overall', 'all']:
+    if demographic in ["overall", "all"]:
         color_keys = cod_groups
     else:
         demo_column = LOOKUP["demo_to_col"].get(demographic, demographic)
         color_keys = va_df[demo_column].unique().tolist()
-    color_dict = {color_key: LOOKUP["color_list"][i] for i, color_key in enumerate(color_keys)}
-    
+    color_dict = {
+        color_key: LOOKUP["color_list"][i] for i, color_key in enumerate(color_keys)
+    }
+
     # build subplots incrementally and store them in a list of dictionaries
     subplots = []
     legend_names = set()
     if va_df.size > 0:
 
         # create 1 subplot per group and store data
-        for i, cod_group in enumerate(cod_groups):
-            
-            # default values for group's height and traces. 
+        for _, cod_group in enumerate(cod_groups):
+
+            # default values for group's height and traces.
             group_height, group_traces = 1, []
-            
+
             # filter va data down to only group of interest
-            cod_data = cod_group_data(va_df, cod_group, N=N)
-            
+            cod_data = cod_group_data(va_df, cod_group, n=n)
```

Comment:
It looks like the `cod_group_data` function is now missing one positional argument (specifically, cod_groups). It seems like this happens because `cod_group_data` is no longer passed the default of argument of `pd.DataFrame()` for `cod_groups`

Calling `cod_group_data(va_df, cod_group, None, n=n)` here worked for me, but I'm sure there are other ways to address this.

Reply:
Implemented suggestion via 57197466361aaa9fea4eaa46ce25a795bc86ea40